client/actions/client-track.js

```diff
@@ -0,0 +1,48 @@
+'use server'
+
+import axios from'axios';
+
+const getAppointment = async (formEmail, formPhone) => {
+    console.log('running get appointment');
+    try {
+        const data = await axios({
+            url: 'http://localhost:9000/client-track',
+            method: 'GET',
+            headers: {"Content-Type": "application/json"},
+            params: {email: formEmail, phone: formPhone}
+        });
+        console.log('returns', data);
+        console.log('from action', data.data.data);
+        return data.data.data;
+    } catch(err) {
+        console.log(err.response.data);
+        return err.response.data;
+    }
+}
+
+const cancelAppointment = async (id) => {
+    console.log('deleting appointment');
+    try {
+        const res = await axios({
+            url: `http://localhost:9000/client-track/details/${id}`,
+            method: 'DELETE',
+            headers: {"Content-Type": "application/json"},
+        })
+
+        if(res.status === 204) {
+            console.log('Delete Successful');
+            return res.status;
```

Comment:
Similar concern about polymorphic returns. I don't think it's intuitive that this returns either a numeric code or a boolean.